pkg/api/convertertoapi.go

```diff
@@ -562,17 +548,19 @@ func convertV20170701OrchestratorProfile(v20170701cs *v20170701.OrchestratorProf
 
 	switch api.OrchestratorType {
 	case Kubernetes:
-		switch v20170701cs.OrchestratorRelease {
+		release, _ := common.GetReleaseFromVersion(v20170701cs.OrchestratorVersion)
+		switch release {
 		case KubernetesRelease1Dot8, KubernetesRelease1Dot7, KubernetesRelease1Dot6, KubernetesRelease1Dot5:
-			api.OrchestratorRelease = v20170701cs.OrchestratorRelease
+			api.OrchestratorRelease = release
```

Comment:
We don't need OrchestratorRelease in unversioned API. Release (if someone needs it) can be derived from OrchestratorVersion. Also, have both OrchestratorRelease  and OrchestratorVersion is prone to bugs

Reply:
As latest discussion, we don't change internal model.